soccer/gameplay/skills/capture.py

```diff
@@ -99,7 +99,8 @@ def on_enter_course_approach(self):
 
     def execute_course_approach(self):
         # don't hit the ball on accident
-        self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        #self.robot.set_avoid_ball_radius(Capture.CourseApproachAvoidBall)
+        self.robot.disable_avoid_ball()
```

Comment:
I agree that our capture is pretty crappy for moving balls.  With out current strategy of getting on the other side of it to catch it, we definitely need the avoid ball line here.  I do think it would be worthwhile to look into going directly for it, but in the meantime we should keep the avoid.